sdk/go/github/actionsOrganizationSecret.go

```diff
@@ -14,90 +14,6 @@ import (
 
 // ## Example Usage
 //
-// <!--Start PulumiCodeChooser -->
-// ```go
-// package main
-//
-// import (
-//
-//	"github.com/pulumi/pulumi-github/sdk/v6/go/github"
-//	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
-//
-// )
-//
-//	func main() {
-//		pulumi.Run(func(ctx *pulumi.Context) error {
-//			_, err := github.NewActionsOrganizationSecret(ctx, "exampleSecretActionsOrganizationSecret", &github.ActionsOrganizationSecretArgs{
-//				SecretName:     pulumi.String("example_secret_name"),
-//				Visibility:     pulumi.String("private"),
-//				PlaintextValue: pulumi.Any(_var.Some_secret_string),
-//			})
-//			if err != nil {
-//				return err
-//			}
-//			_, err = github.NewActionsOrganizationSecret(ctx, "exampleSecretIndex/actionsOrganizationSecretActionsOrganizationSecret", &github.ActionsOrganizationSecretArgs{
-//				SecretName:     pulumi.String("example_secret_name"),
-//				Visibility:     pulumi.String("private"),
-//				EncryptedValue: pulumi.Any(_var.Some_encrypted_secret_string),
```

Comment:
Suspect it fell out because of undefined _var.Some_encrypted_secret_string - checking

Reply:
Hmm

```
{
  "ExampleName": "#/resources/github:index/actionsOrganizationSecret:ActionsOrganizationSecret",
  "OriginalHCL": "resource \"github_actions_organization_secret\" \"example_secret\" {\n  secret_name     = \"example_secret_name\"\n  visibility      = \"private\"\n  plaintext_value = var.some_secret_string\n}\n\nresource \"github_actions_organization_secret\" \"example_secret\" {\n  secret_name     = \"example_secret_name\"\n  visibility      = \"private\"\n  encrypted_value = var.some_encrypted_secret_string\n}\n",
  "ConversionResults": {
    "csharp": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "go": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "java": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "python": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "typescript": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    },
    "yaml": {
      "FailureSeverity": 2,
      "FailureInfo": "Duplicate resource \"github_actions_organization_secret\" configuration: A github_actions_organization_secret resource named \"example_secret\" was already declared at /e27.tf:1,1-63. Resource names must be unique per type in each module.",
      "Program": "",
      "TranslationCount": 1
    }
  }
}

```

Reply:
https://github.com/pulumi/pulumi-converter-terraform/issues/146